variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
```

Comment:
I think that "Value of Lost Load" is the more common term here, so I'd suggest to use this as the variable name

Reply:
Ok, I got it.